backend.native/cli.bc/src/org/jetbrains/kotlin/cli/bc/K2NativeCompilerArguments.kt

```diff
@@ -103,6 +103,14 @@ class K2NativeCompilerArguments : CommonCompilerArguments() {
     @Argument(value = "-Xenable", deprecatedName = "--enable", valueDescription = "<Phase>", description = "Enable backend phase")
     var enablePhases: Array<String>? = null
 
+    @Argument(
+            value = "-Xexport-library",
+            valueDescription = "<path>",
+            description = "Path to the library to be included into produced framework API\n" +
+                    "Must be the path of a library passed with '-library'"
```

Comment:
Can platform libs or stdlib be exported this way?

Reply:
Can it be the name, not path?
Gradle plugin support for that shall be done as well.

Reply:
> Can platform libs or stdlib be exported this way?

Yes.

> Can it be the name, not path?

No, because this would require the feature to be tightly coupled with library resolution machinery.

> Gradle plugin support for that shall be done as well.

AFAIK, Gradle provides absolute paths for libraries when invoking the compiler.